sdk/communication/azure-communication-sms/src/test/java/com/azure/communication/sms/SmsClientTests.java

```diff
@@ -165,6 +166,21 @@ public void sendTwoMessages(HttpClient httpClient) {
         assertHappyPath(secondResponse);
     }
 
+    @ParameterizedTest
+    @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    public void repeatability(HttpClient httpClient) {
+        PipedOutputStream osPipe = new PipedOutputStream();
+        // Arrange
+        SmsClientBuilder builder = getSmsClientUsingConnectionString(httpClient);
+        client = setupSyncClient(builder, "sendTwoMessagesSync");
+
+        // Action & Assert
+        Response<Iterable<SmsSendResult>> response = client.sendWithResponse(FROM_PHONE_NUMBER, Arrays.asList(TO_PHONE_NUMBER, TO_PHONE_NUMBER), MESSAGE, null, Context.NONE);
+        String bodyRequest = new String(response.getRequest().getBody().blockLast().array());
+        assertTrue(bodyRequest.contains("repeatabilityRequestId"));
+
```

Comment:
We should probably add a check for the RepeatabilityFirstSent as well

Reply:
we should probably also validate that repeatabilityRequestId value follows a guid format if were going to make a test for it

Reply:
Just validated the words repeatabilityRequestId  and repeatabilityFirstSent